test/archethic/p2p/geo_patch_test.exs

```diff
@@ -40,5 +56,9 @@ defmodule ArchEthic.P2P.GeoPatchTest do
     assert "A1A" == GeoPatch.from_ip({15, 62, 246, 57})
     assert "021" == GeoPatch.from_ip({109, 164, 214, 168})
     assert "8C0" == GeoPatch.from_ip({1, 2, 3, 4})
+    assert "F1F" == GeoPatch.from_ip({1, 1, 1, 1})
```

Comment:
Is this distance good ? I mean it's almost close

Reply:
Yes, first digit F means that longitude is >= -22.5 and < 0, when going in positive we return to 0. i.e 0 is >= 0 and < 22.5.
So just after FFF we have 000.

Reply:
Ok .
@imnik11 can you have a last check ?